1 Presentation layer/CandidateEvaluator.Server/Controllers/QuestionController.cs

```diff
@@ -0,0 +1,65 @@
+﻿
+using CandidateEvaluator.Contract.Commands.Question;
+using CandidateEvaluator.Contract.Dispatchers;
+using CandidateEvaluator.Contract.Queries.Question;
+using CandidateEvaluator.Server.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace CandidateEvaluator.Server.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    public class QuestionController : ControllerBase
+    {
+        private IDispatcher _dispatcher;
+
+        public QuestionController(IDispatcher dispatcher)
+        {
+            _dispatcher = dispatcher;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Create([FromBody] CreateQuestion command)
+        {
+            command.OwnerId = HttpContext.GetUser().Oid;
+            var created = await _dispatcher.SendAsync(command);
+            return CreatedAtAction(nameof(Create), created);
```

Comment:
Here should be `Get` instead of `Create`. `CreatedAtAction` produces `201` status code with URI to get created resource.

Reply:
Done